pkg/restapi/operation/models.go

```diff
@@ -38,10 +38,15 @@ type authProviders struct {
 	Providers []authProvider `json:"authProviders"`
 }
 
+type OidcProviderIconURL struct {
+	En string
+	Fr string
+}
+
 type authProvider struct {
-	ID            string `json:"id"`
-	Name          string `json:"name"`
-	SignUpLogoURL string `json:"signUpLogoUrl"`
-	SignInLogoURL string `json:"signInLogoUrl"`
-	Order         int    `json:"order"`
+	ID            string              `json:"id"`
+	Name          string              `json:"name"`
+	SignUpIconURL OidcProviderIconURL `json:"signUpIconUrl"`
+	SignInIconURL OidcProviderIconURL `json:"signInIconUrl"`
```

Comment:
Can we use a map[string]string instead of a struct here? Using a map will help us scale when we add support for new languages, and we don't need to update the struct each time.